pytorch/Dockerfile

```diff
@@ -109,6 +109,57 @@ RUN wget -q  --no-check-certificate https://raw.githubusercontent.com/oneapi-src
     wget -q  --no-check-certificate https://raw.githubusercontent.com/intel/neural-compressor/master/docker/third-party-programs-pytorch.txt -O /licensing/third-party-programs-pytorch.txt && \
     wget -q  --no-check-certificate https://raw.githubusercontent.com/intel/neural-compressor/master/LICENSE -O /licensing/LICENSE
 
+FROM multinode AS hf-genai
+
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get clean autoclean && \
+    apt-get autoremove -y && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN apt-get update -y && \
+    apt-get install -y --no-install-recommends --fix-missing \
+    google-perftools \
+    libjemalloc2 \
+    libomp-dev \
+    numactl \
+    git
+
+COPY hf-genai-requirements.txt .
+
+RUN python -m pip install --no-cache-dir -r hf-genai-requirements.txt
+
+# Install transformers from source to provide support for transformers example scripts
+RUN git clone https://github.com/huggingface/transformers.git && \
```

Comment:
🤔  I don't think it's necessary to build transformers from source. It could go in `hf-genai-requirements.txt`. If we aren't building from source, then the only reason you are cloning the repo is for the language-modeling/requirements.txt. I compared their requirements.txt with ours and the only thing that I'm seeing theirs have that ours doesn't have is `scikit-learn`, so it seems like it would be easier to just add that to our requirements.txt and then we can remove the cloning.

Reply:
Yeah that's a better idea. Thanks

Reply:
I ended up consolidating it into hf-genai-requirements.txt